playbooks/files/infrastructure/vm.tf

```diff
@@ -20,7 +20,7 @@ resource "opentelekomcloud_compute_instance_v2" "test_host" {
     opentelekomcloud_compute_secgroup_v2.public_ssh
   ]
   security_groups = [
-    opentelekomcloud_compute_secgroup_v2.public_ssh.id
+    opentelekomcloud_compute_secgroup_v2.public_ssh.name
```

Comment:
are you sure?

Reply:
Otherwise terraform modifies ECS's sec groups every apply